openshift_pipelines/installing-pipelines.adoc

```diff
@@ -0,0 +1,19 @@
+[id="installing-pipelines"]
+= Installing OpenShift Pipelines
+include::modules/common-attributes.adoc[]
+include::modules/openshift-pipelines-attributes.adoc[]
+:context: installing-pipelines
+
+toc::[]
+
+// Installing Openshift Pipelines Operator
```

Comment:
Should we include installation via CLI similar to these?
https://openshift.github.io/pipelines-docs/docs/0.10.5/proc_installing-pipelines-operator-using-the-cli.html

Reply:
@siamaksade - This section that I am pointing to from the Installation Pipelines section, covers both - installation using webconsole, and installation using CLI:
https://installing-pipelines--ocpdocs.netlify.com/openshift-enterprise/latest/operators/olm-adding-operators-to-cluster.html#olm-adding-operators-to-a-cluster

Reply:
added the procedure